src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -81,6 +81,17 @@ public UniquePersonList(UniquePersonList source) {
         return Collections.unmodifiableList(internalList);
     }
 
+    /**
+     * Returns an unmodifiable java List view with elements cast as immutable {@link ReadOnlyPerson}s.
+     * For use with other methods/libraries.
+     * Any changes to the internal list/elements are immediately visible in the returned list.
+     */
+    public List<ReadOnlyPerson> sortedListView() {
+        //internalList.sort(); <--how to make this line work instead of using Collections.sort()? :(
```

Comment:
https://docs.oracle.com/javase/8/docs/api/java/util/List.html
You have to pass in a comparator in this case. However, we usually just use Collections.sort/ Arrays.sort